gtsam/geometry/tests/testPose3.cpp

```diff
@@ -145,6 +145,75 @@ TEST(Pose3, Adjoint_full)
   EXPECT(assert_equal(expected3, Pose3::Expmap(xiprime3), 1e-6));
 }
 
+/* ************************************************************************* */
+// Check Adjoint numerical derivatives
+TEST(Pose3, Adjoint_jacobians)
+{
+  Matrix6 actualH1, actualH2, expectedH1, expectedH2;
+  std::function<Vector6(const Pose3&, const Vector6&)> Adjoint_proxy =
+      [&](const Pose3& T, const Vector6& xi) {
+        Vector6 res1 = T.AdjointMap() * xi, res2 = T.Adjoint(xi);
+        EXPECT(assert_equal(res1, res2));
```

Comment:
It's not super clean to add a check inside a functional. You want to be able to isolate issues with results and Jacobians.

Reply:
Ya this has bitten me before - I'll switch to copy-pasting this line

Reply:
done